pkg/resmgr/nri.go

```diff
@@ -100,7 +104,9 @@ func (p *nriPlugin) onClose() {
 }
 
 func (p *nriPlugin) Configure(cfg, runtime, version string) (stub.EventMask, error) {
-	p.dump("NRI-Configure", "data", cfg)
+	event := "Configure"
+	p.dump(request, event, cfg)
```

Comment:
passing cfg seems unnecessary here, because we are not doing anything with it in the dump(). Perhaps we either remove it from here and extend the dump (`event == Configure` case). WDYT?